book/src/pong_tutorial/pong_tutorial_02.md

```diff
@@ -39,20 +39,89 @@ stops complaining about unused imports. In `pong.rs` we'll need these use
 statements to make it through this chapter:
 
 ```rust,ignore
-use amethyst::assets::Loader;
-use amethyst::core::cgmath::Vector3;
+use amethyst::assets::{AssetStorage, Loader};
+use amethyst::core::cgmath::{Vector3, Matrix4};
 use amethyst::core::transform::{GlobalTransform, Transform};
 use amethyst::ecs::prelude::{Component, DenseVecStorage};
+use amethyst::input::{is_close_requested, is_key};
 use amethyst::prelude::*;
-use amethyst::renderer::{Camera, Event, KeyboardInput, Material, MaterialDefaults, MeshHandle,
-                         PosTex, VirtualKeyCode, WindowEvent};
+use amethyst::renderer::{
+    Camera, Event, PngFormat, Projection, Sprite, Texture, TextureHandle,
+    VirtualKeyCode, WithSpriteRender,
+};
 ```
 
+## Get around the World
+
+First, in `pong.rs`, let's add a new method to our State implementation: `on_start`.
+This method is called, as you probably guessed, on the start of the State.
```

Comment:
on the start of the State. -> when the State starts

(the important part is the "when")